src/cpp/session/modules/SessionFind.cpp

```diff
@@ -1643,24 +1752,25 @@ core::Error Replacer::completeReplace(const boost::regex& searchRegex,
       return error;
    }
 
-   temp.insert(0, pLine->substr(0, matchOn));
-   std::string endOfString = pLine->substr(matchOff).c_str();
-   size_t replaceMatchOff;
-   if (endOfString.empty())
-      replaceMatchOff = temp.length();
-   else
-      replaceMatchOff = temp.find(endOfString);
+   newLine.insert(0, begin);
+   *pReplaceMatchOff = newLine.length() - end.length();
+   *pLine = newLine;
+/*
```

Comment:
Do we need this commented code? (feels like it should just be removed)

Reply:
You're right, just me making sure I had the right math. I've removed it.